README.md

```diff
@@ -173,16 +167,16 @@ The following permissions are required or recommended.
 
 ### OSS Kubernetes
 
-Please check out [Using KHI with OSS Kubernetes Clusters - Example with Loki](./docs/en/oss.md).
+Read [Using KHI with OSS Kubernetes Clusters - Example with Loki](./docs/en/oss.md).
 
 ## User Guide
 
-Please check our [user guide](./docs/en/user-guide.md).
+Read [user guide](./docs/en/user-guide.md).
 
-## Contribute
+## Development Contribution Guide
 
-If you'd like to contribute to the project, please read our [Contributing guide](./docs/contributing.md).
+If you'd like to contribute to code KHI, read [Contribution Guide](./docs/contributing.md).
```

Comment:
Why did we limit this scope to code? There are many area of contribution without coding anything.